_posts/2017/2017-11-01-internal-workshops.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: post
+title: "Gradle Workshop"
+date: "2017-11-01 22:35:13"
+image: blog-header/workshop.jpg
+image-position: bottom
+categories: agile
+authors: ["Thomas"]
+---
+
+## Introduction
+
+Recently I conducted my fourth Gradle workshop, and for the first time at ePages.<br>
```

Comment:
Remove "< br >" everywhere it appears.

Reply:
We've had this discussion, have to use <br> because two spaces at the end of the line will not be accepted by the linter.
What it does is a line break without creating a new paragraph.

Reply:
By the way in order to type a tag like this in a comment for example, use HTML-escaping:

&amplt; for the "less than" sign (the opening bracket), and &ampgt; for the "greater than" sign (the closing bracket)

Reply:
Explanation:

`&lt;br&gt;` is the HTML tag for a linebreak
When Markdown creates a HTML document from this, it would replace two spaces at the end of a line